.github/workflows/release.yml

```diff
@@ -0,0 +1,91 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
@iskey should we also have the release job to generate tags or should we push tag manually? If we need a full automation, it would be good to have tags, github release and docker image created by one single job. (similar to what we have in xpanse, ui, terraform-boot)

Reply:
Hi @swaroopar , yes, generating tags is better, I will take that.